packages/pipeline/src/scripts/pull_ddex_orderbook_snapshots.ts

```diff
@@ -0,0 +1,62 @@
+// tslint:disable:no-console
+import { BigNumber } from '@0x/utils';
+import axios from 'axios';
+import * as R from 'ramda';
+import { Connection, ConnectionOptions, createConnection, EntityManager } from 'typeorm';
+
+import { DdexActiveMarketsResponse, DdexMarket, DdexOrder, DdexOrderbookResponse } from '../data_sources/ddex';
+import { TokenOrderbookSnapshot as TokenOrder } from '../entities';
+import * as ormConfig from '../ormconfig';
+import { parseDdexOrders } from '../parsers/ddex_orders';
+import { handleError } from '../utils';
+
+const DDEX_BASE_URL = 'https://api.ddex.io/v2';
```

Comment:
I was talking to @xianny about this and I think it is more appropriate to move URLs into the *data_sources* directory. In most cases, whenever we send an HTTP request, the response format is tied to the URL. This means that you can't really change the URL without making changes to the *data_sources* directory. They are tightly coupled, so we should keep them in the same place to make that coupling more obvious. What do you think?

Planning to make the same change to our other scripts eventually.